Tests/OrdersTests/OrderData.swift

```diff
@@ -0,0 +1,126 @@
+import Fluent
+import struct Foundation.UUID
+import Orders
+import Vapor
+
+final class OrderData: OrderDataModel, @unchecked Sendable {
+    static let schema = OrderData.vddMMyyyy.schemaName
+    
+    @ID(key: .id)
+    var id: UUID?
+
+    @Field(key: OrderData.vddMMyyyy.title)
+    var title: String
+    
+    @Parent(key: OrderData.vddMMyyyy.orderID)
+    var order: Order
+
+    init() { }
+
+    init(id: UUID? = nil, title: String) {
+        self.id = id
+        self.title = title
+    }
+    
+    func toDTO() -> OrderDataDTO {
+        .init(
+            id: self.id,
+            title: self.$title.value
+        )
+    }
+}
+
+struct OrderDataDTO: Content {
+    var id: UUID?
+    var title: String?
+    
+    func toModel() -> OrderData {
+        let model = OrderData()
+        
+        model.id = self.id
+        if let title = self.title {
+            model.title = title
+        }
+        return model
+    }
+}
+
+struct CreateOrderData: AsyncMigration {
+    func prepare(on database: any Database) async throws {
+        try await database.schema(OrderData.vddMMyyyy.schemaName)
+            .id()
+            .field(OrderData.vddMMyyyy.title, .string, .required)
+            .field(OrderData.vddMMyyyy.orderID, .uuid, .required, .references(Order.schema, .id, onDelete: .cascade))
+            .create()
+    }
+
+    func revert(on database: any Database) async throws {
+        try await database.schema(OrderData.vddMMyyyy.schemaName).delete()
+    }
+}
+
+extension OrderData {
+    enum vddMMyyyy {
```

Comment:
😅

Reply:
This is a best practice I read in the Kodeco book for versioning APIs

Reply:
`ddMMYYYY` should be replaced with an actual date 😅 

So it should be `v20240723` etc

Reply:
Oh, I thought this was the documentation. In the tests I could just name it FieldKeys